github-pr

```diff
@@ -12,11 +12,19 @@ def _check_required_fields(required, **args):
             sys.exit(1)
 
 
+def _return_status(commits):
+    commit_statuses = []
+    for commit in commits:
+        for commit_status in commit.get_statuses():
+            commit_statuses.append(commit_status.state)
+    return commit_statuses[-1]
```

Comment:
(github_env)mbabineau@matt-bs-mbp:~/dev/github-pr$ github-pr list -r dataxu/jenkins-jobs --token=${GIT_TOKEN} -n 339
Traceback (most recent call last):
  File "/Users/mbabineau/dev/github-pr/github_env/bin/github-pr", line 5, in <module>
    pkg_resources.run_script('github-pr==1.0.0', 'github-pr')
  File "/Users/mbabineau/dev/github-pr/github_env/lib/python2.7/site-packages/pkg_resources.py", line 534, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/Users/mbabineau/dev/github-pr/github_env/lib/python2.7/site-packages/pkg_resources.py", line 1441, in run_script
    exec(script_code, namespace, namespace)
  File "/Users/mbabineau/dev/github-pr/github_env/lib/python2.7/site-packages/github_pr-1.0.0-py2.7.egg/EGG-INFO/scripts/github-pr", line 210, in <module>

  File "/Users/mbabineau/dev/github-pr/github_env/lib/python2.7/site-packages/github_pr-1.0.0-py2.7.egg/EGG-INFO/scripts/github-pr", line 91, in github_list_prs

  File "/Users/mbabineau/dev/github-pr/github_env/lib/python2.7/site-packages/github_pr-1.0.0-py2.7.egg/EGG-INFO/scripts/github-pr", line 29, in _print_pr

  File "/Users/mbabineau/dev/github-pr/github_env/lib/python2.7/site-packages/github_pr-1.0.0-py2.7.egg/EGG-INFO/scripts/github-pr", line 17, in _return_status

  File "/Users/mbabineau/dev/github-pr/github_env/lib/python2.7/site-packages/PyGithub-1.25.2-py2.7.egg/github/PaginatedList.py", line 40, in __getitem__
    return self.__elements[index]
IndexError: list index out of range